packages/server/graphql/types/AtlassianIntegration.ts

```diff
@@ -8,7 +8,6 @@ import {
   GraphQLString
 } from 'graphql'
 import ms from 'ms'
-import AtlassianAuth from '../../database/types/AtlassianAuth'
```

Comment:
I changed the type we use from the old Rethink model to the type used by PG. In PG type both access and refresh token is not nullable so it simplifies the code.